modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -698,6 +698,11 @@ namespace detail
         static const char* tag() { return "gapi.use_only"; }
     };
 } // namespace detail
+
+#ifdef OPENCV_BINDINGS_PARSER
```

Comment:
Should be `#if OPENCV_BINDINGS_PARSER`

Reply:
Are u sure ?     
`/build/precommit_linux64/opencv/modules/gapi/include/opencv2/gapi/gkernel.hpp:702:5: warning: "OPENCV_BINDINGS_PARSER" is not defined [-Wundef]`

Reply:
Reverted